<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>landmarkRegistration</class>
 <widget class="QDialog" name="landmarkRegistration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Landmark Registration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Volume Sync</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="syncSTButton">
        <property name="text">
         <string>Sync S-&gt;T</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="syncTSButton">
        <property name="text">
         <string>Sync T-&gt;S</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QPushButton" name="newPointsButton">
     <property name="text">
      <string>New Points</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="newPointsName"/>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Point Placement</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="sourceRadio">
        <property name="text">
         <string>Source</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="targetRadio">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Target</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="colorMatchButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Color Match</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="refreshButton">
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Source Volume</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="volSource"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Source Points</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="setSource"/>
       </item>
       <item>
        <widget class="QTableWidget" name="setOneTable">
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>53</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>53</number>
         </attribute>
         <column>
          <property name="text">
           <string>X</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Y</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Z</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Color</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Target Volume</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="volTarget"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Target Points</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="setTarget">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="setTwoTable">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>53</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>53</number>
         </attribute>
         <column>
          <property name="text">
           <string>X</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Y</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Z</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Color</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Registration Method</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="registerBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="registerButton">
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
